src/main/java/seedu/recipe/logic/commands/EditCommand.java

```diff
@@ -17,12 +17,7 @@
 import seedu.recipe.commons.util.CollectionUtil;
 import seedu.recipe.logic.commands.exceptions.CommandException;
 import seedu.recipe.model.Model;
-import seedu.recipe.model.recipe.Description;
-import seedu.recipe.model.recipe.Ingredient;
-import seedu.recipe.model.recipe.Recipe;
-import seedu.recipe.model.recipe.Step;
-import seedu.recipe.model.recipe.Title;
-
+import seedu.recipe.model.recipe.*;
```

Comment:
I think using * may not pass the CheckStyle but if you think it's good then alright.